index.d.ts

```diff
@@ -840,6 +840,12 @@ declare namespace Eris {
   }
 
   // Guild
+  interface GuildChannelPosition {
+    id: string;
+    position: number;
+    lockPermissions?: boolean;
+    parentID?: string;
+  }
```

Comment:
This can probably be called ChannelPosition btw, and should go under the Channel category - please also put this in the relevant alphabetical location. I know Discord calls it GuildChannelPosition, but it's obvious that it belongs in a guild

Reply:
Changed it as requested